app/views/browse/deployment.html

```diff
@@ -91,7 +91,10 @@
                   <uib-tab ng-if="metricsAvailable" heading="Metrics" active="selectedTab.metrics">
                     <!-- Use ng-if to remove the metrics directive when the tab is not active so
                          we don't update in the background. -->
-                    <metrics ng-if="selectedTab.metrics" deployment="deployment"></metrics>
+                    <deployment-metrics
```

Comment:
so when the monitoring PR merges it will need to use this directive instead, for deployments